docs/guides/genomics-platform/requesting-data/about-our-data.md

```diff
@@ -23,19 +22,38 @@ We provide gVCF files produced by the [Microsoft Genomics Service][msgen]. gVCF
 
 ### Somatic VCF files
 
-Somatic VCF files contain HG38 based SNV/Indel variant calls from the St. Jude somatic variant analysis pipeline as follows. Broadly speaking:
+Somatic VCF files available on St. Jude Cloud have been generated when possible for tumor samples where a matching normal sample has also been sequenced. These include samples from the Pediatric Cancer Genome Project (PCGP), Clinical Pilot, Genomes 4 Kids (G4K), and Real-time Clinical Genomics (RTCG) datasets (https://platform.stjude.cloud/data/cohorts/pediatric-cancer). These somatic VCF files contain HG38 coordinate SNV/Indel variant calls generated by the St. Jude somatic variant analysis pipeline outlined below:
 
-1. Reads were aligned to HG19 using [bwa backtrack][bwa] (`bwa aln` + `bwa sampe`) using default parameters.
+1. WGS and WES reads were aligned to HG19 using [bwa backtrack][bwa] (`bwa aln` + `bwa sampe`) using default parameters.
 2. Post processing of aligned reads was performed using [Picard][picard] `CleanSam` and `MarkDuplicates`.
 3. Variants were called using the [Bambino][bambino-paper] variant caller (you can download Bambino [here][bambino-download] or by navigating to the [Zhang Lab page][bambino-program] where the  "Bambino package" is listed as a dependency under the CONSERTING section).
 4. Variants were post-processed using an in-house post-processing pipeline that cleans and annotates variants. This pipeline is not currently publicly available.
-5. Variants were manually reviewed by analysts and published with [the relevant Pediatric Cancer Genome Project (PCGP) paper][pcgp-landing-page].
-6. Post-publication, variants were lifted over to HG38 (the original HG19 coordinates are stored in the `HG19` INFO field.).
+5. Depending on cohort, either all variants were manually reviewed by analysts or a subset comprising all coding variants were manually reviewed.
```

Comment:
Lets be more specific here... and specify which cohort/dataset was treated in which way....